wallet/src/lmdb_wallet.rs

```diff
@@ -199,6 +199,17 @@ where
 		&mut self.w2n_client
 	}
 
+	/// return the commit, if enabled, None otherwise
+	fn cached_commit(&mut self, amount: u64, id: &Identifier) -> Result<Option<String>, Error> {
```

Comment:
Is this maybe mis-named or am I misunderstanding what is going on here?
Its not actually cached in here?